src/Microsoft.VisualStudio.LanguageServices.Razor/Editor/DefaultVisualStudioDocumentTracker.cs

```diff
@@ -27,7 +28,7 @@ internal class DefaultVisualStudioDocumentTracker : VisualStudioDocumentTracker
         private ProjectSnapshot _project;
         private string _projectPath;
 
-        public override event EventHandler ContextChanged;
+        public override event EventHandler<ContextChangeEventArgs> ContextChanged;
```

Comment:
This is a breaking change. Does WTE use this?

Reply:
Yes. When we insert these changes we'll need to update WTE anyways.